crates/contracts/src/array_tr_storage/mod.rs

```diff
@@ -68,15 +68,15 @@ pub fn execute_array_tr_storage_program(
 /// The unspendable internal key specified in BIP-0341.
 #[rustfmt::skip] // mangles byte vectors
 #[must_use]
-pub fn unspendable_internal_key() -> secp256k1::XOnlyPublicKey {
+pub fn unlimited_storage_unspendable_internal_key() -> secp256k1::XOnlyPublicKey {
```

Comment:
In 0830d7b45d6f1a37c6c3785563018c6b349eb38f:

This rename doesn't make sense. There is nothing specific to this contract about the BIP341 unspendable key.